package.json

```diff
@@ -18,7 +18,10 @@
     "ava": "ava --verbose --extension ts",
     "ts-node": "ts-node",
     "dist": "npm run clean && tsc && jq \"del (.files)\" < package.json > dist/package.json && shx cp src/puppet-web/*.js dist/src/puppet-web/",
-    "doc": "npm run dist && echo '# Wechaty v'$(jq -r .version package.json)' Documentation\n' > docs/index.md && jsdoc2md dist/src/{wechaty,room,contact,friend-request,message}.js dist/src/puppet-web/friend-request.js>> docs/index.md",
+    "doc": "npm run dist && echo '# Wechaty v'$(jq -r .version package.json)' Documentation\n' > docs/index.md && npm run doc-api && npm run doc-readme",
```

Comment:
run `npm run doc`, output:

```cmd
E:\CodeWork\fork\wechaty>npm run doc

> wechaty@0.8.231 doc E:\CodeWork\fork\wechaty
> npm run dist && echo '# Wechaty v'$(jq -r .version package.json)' Documentation
' > docs/index.md && npm run doc-api && npm run doc-readme


> wechaty@0.8.231 dist E:\CodeWork\fork\wechaty
> npm run clean && tsc && jq "del (.files)" < package.json > dist/package.json && shx cp src/puppet-web/*.js dist/src/puppet-web/


> wechaty@0.8.231 clean E:\CodeWork\fork\wechaty
> shx rm -fr dist/*

'# Wechaty v'$(jq -r .version package.json)' Documentation
```

Under Windows, \n will parse the error, causing the execution to be interrupted

In addition, echo command in the windows and linux system implementation of the results are different 😂 

```cmd
E:\CodeWork\fork\wechaty>echo '# Wechaty v'$(jq -r .version package.json)' Documentation'
'# Wechaty v'$(jq -r .version package.json)' Documentation'

E:\CodeWork\fork\wechaty>bash
li@BinWork:/mnt/e/CodeWork/fork/wechaty$ echo '# Wechaty v'$(jq -r .version package.json)' Documentation'
# Wechaty v0.8.231 Documentation
```

Reply:
System differences, there is no way

Generating readme very well